packages/apidom-ls/src/config/common/schema/lint/missing-core-fields-oas-3-0-asyncapi.ts

```diff
@@ -0,0 +1,91 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
```

Comment:
Just to make naming consistent with the import symbol:

`missing-core-fields-oas-3-0-asyncapi.ts` -> `missing-core-fields-openapi-3-0-asyncapi-2.ts`